gae/room/staffposition_api.py

```diff
@@ -0,0 +1,76 @@
+"""API for StaffPositions and their (hourly,mileage) info."""
+
+from google.appengine.ext import ndb
+
+from protorpc import message_types
+from protorpc import messages
+from protorpc import remote
+from protorpc.wsgi import service
+
+import base_api
+import ndb_models
+import protorpc_messages
```

Comment:
Is this import needed?

Reply:
It's only used 1 time for staffposition_read
Should I "import SimpleId from protorpc_messages"?